ir/json_loader.h

```diff
@@ -223,10 +224,35 @@ class JSONLoader {
         load(::P4::get(obj, "value"), value), v = std::move(value);
     }
 
+    template <int N, class Variant>
+    std::enable_if_t<N == std::variant_size_v<Variant>, void> unpack_variant(
+        const JsonObject *, int /*target*/, Variant & /*variant*/) {
+        BUG("Error traversing variant during load");
+    }
+
+    template <int N, class Variant>
+        std::enable_if_t <
+        N<std::variant_size_v<Variant>, void> unpack_variant(const JsonObject *obj, int target,
```

Comment:
Wow, the formatter is quite confused here. Putting brackets around the "less than" might fix this.

Reply:
Oh, yes. It does not expect such logic here :)